src/animations/bar-progress.scss

```diff
@@ -0,0 +1,46 @@
+$initial-size: 20%;
+$height: 12px;
+$radius: 10px;
+
+@keyframes bar-progress {
+  0% {
+    width: $initial-size;
+    left: 0;
```

Comment:
Hey Caio, only `transform`, and `opacity` are supposed to be here. The left and width property forces a repaint which is something the animations in this repo try to avoid. I love the animations though, the swing container was my favorite, if you can re-work them to use the two properties mentioned above then I'll happily merge them in.